README.md

```diff
@@ -1,87 +1,62 @@
 # Xamarin.ZoomSDKBinding
  
-* Android: Meeting SDK Version: 5.11.10.8019
+* Android: Meeting SDK Version: 5.12.2.9109
  
 * iOS: 
  	* MobileRTC Version: 5.11.10.4556
  	* MobileRTC Screen Share Version: Not implemented yet, PRs are welcome! 
  
 ## Android Gotchas
 * Requires your android app to compile for Android 12
-* Only tested with Zoom Custom UI. Default UI may work, but it is not tested.
-* Requires adding two dummy resource files in the Android Resources folders to compensate for the lack of a compatible version of the exoplayer binding library for xamarin. They are seeking open source contributions at [their repo](https://github.com/Baseflow/ExoPlayerXamarin) for the upgrade on [this issue](https://github.com/Baseflow/ExoPlayerXamarin/issues/116#issuecomment-884687316), which would remove the need for the dummy resource files. The missing resources do not affect apps running the custom ui, but if you are attempting to use the default UI, you might conceivably run into problems. (Did I mention default UI is not tested?!)

```

Comment:
@icalderond Is this removed because you added exoplayer slim? If so that is great! Do we still need to include the dummy resource files in the Xamarin.Android project to build?

Reply:
It is correct. I added exoplayer slim and it not was required include dummy resources.